src/engine/CMakeLists.txt

```diff
@@ -102,6 +102,8 @@ add_subdirectory(${ENGINE_SOURCE_DIR}/shared)
 add_subdirectory(${ENGINE_SOURCE_DIR}/catalog)
 add_subdirectory(${ENGINE_SOURCE_DIR}/server)
 add_subdirectory(${ENGINE_SOURCE_DIR}/router)
+add_subdirectory(${ENGINE_SOURCE_DIR}/kvdb)
+# TODO dependent
```

Comment:
This comment was eliminated on previous Engine commits, and we should keep it deleted.